testsuite/forge-cli/src/main.rs

```diff
@@ -1950,6 +1947,13 @@ fn realistic_env_max_load_test(
     let long_running = duration_secs >= 2400;
 
     let mut success_criteria = SuccessCriteria::new(95)
+        .add_system_metrics_threshold(SystemMetricsThreshold::new(
+            // Check that we don't use more than 18 CPU cores for 10% of the time.
+            MetricsThreshold::new(18.0, 10),
+            // Memory starts around 3GB, and grows around 1.2GB/hr in this test.
+            // Check that we don't use more than final expected memory for more than 10% of the time.
+            MetricsThreshold::new_gb(3.3 + 1.4 * (duration_secs as f64 / 3600.0), 10),
```

Comment:
Should we change the memory metrics to use RSS instead - that way this will be more stable for long running test?

Reply:
we are using container_memory_rss, that increases too.

overall memory grows 3GB/hour